Automattic/woocommerce-payments
# Deposit details - for instant deposits, replace the transaction list with a message

Fixes #7771

#### Changes proposed in this Pull Request









This PR replaces the transactions list on the instant deposit details screen with a message explaining that the transaction history is unavailable for instant deposits.

The "Learn more" link should point to `https://woo.com/document/woopayments/deposits/instant-deposits/#transactions`, with the `#transactions` section of the doc yet to come. See paJDYF-bfU-p2#comment-22039.

![instant deposit details](https://github.com/Automattic/woocommerce-payments/assets/3147296/a8f560ad-7ba4-430b-9296-8b6b1d6c8640)


> [!note]
> Changes to the deposit details summary section will occur as part of the upcoming Instant Deposits project (changes to heading, colours, etc).

#### Testing instructions





* Use a store that has both previously initiated instant and schedules deposits
	* I have a store running that meets this criteria and is running this branch.
	  For credentials, search secret store for `ej-wcpay-usa`.
		* [Scheduled deposit details example](https://ej-wcpay-usa.mystagingwebsite.com/wp-admin/admin.php?page=wc-admin&path=/payments/deposits/details&id=po_1ODzWvCF0hyLh2jp6CzRWpgh)
		* [Instant deposit details example](https://ej-wcpay-usa.mystagingwebsite.com/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Fdeposits%2Fdetails&id=po_1MPIcmCF0hyLh2jpVxedZ0ks)
* View the deposit details screen for:
	* **Paid** scheduled deposit.
		* Ensure the loading state looks correct.
		* Ensure the transaction list is visible as expected.
	* **Paid** instant deposit.
		* Ensure the loading state looks correct.
		* Ensure the transaction list is no longer visible.
		* The message should render as per the design.
		* The link to [Instant Deposits with WooPayments - WooCommerce](https://woo.com/document/woopayments/deposits/instant-deposits/#transactions) should open in a new tab/window.

-------------------


**Post merge**
